pbs.py

```diff
@@ -44,7 +44,7 @@ def make_pbsnodes(orig_file, out_file, write_method):
             pbs_values['core_job_map'] = []
             jobs = block['jobs'].split(',')
             for job, core in get_jobs_cores(jobs):
-                _d = {}
+                _d = dict()

```

Comment:
no difference (not that i know of), is preferred over {} by pycharm in its suggestions. (explicit better than implicit??)